src/main/java/org/folio/circulation/resources/SlipsResource.java

```diff
@@ -354,7 +354,10 @@ private CompletableFuture<Result<StaffSlipsContext>> fetchItemLevelRequests(
 
     final Result<CqlQuery> typeQuery = exactMatch(REQUEST_TYPE_KEY, requestType.getValue());
     final Result<CqlQuery> statusQuery = exactMatch(STATUS_KEY, RequestStatus.OPEN_NOT_YET_FILLED.getValue());
-    final Result<CqlQuery> statusAndTypeQuery = typeQuery.combine(statusQuery, CqlQuery::and);
+    var requestLevelQuery = exactMatch(REQUEST_LEVEL_KEY, RequestLevel.ITEM.getValue());
+    final Result<CqlQuery> statusAndTypeQuery = requestType.equals(RequestType.PAGE) ? typeQuery.combine(statusQuery, CqlQuery::and)
```

Comment:
A couple of questions:
- We're only applying request level filtering to Page request types (pick slips) here. Is this change irrelevant for search slips (Hold requests)?
- I think this line's length exceeds 100

Reply:
Answers for the question --
The issue is raised specifically for print pick slips feature for displaying number of copies and therefore as of now its applicable for only Page request type. 
Second point i will change it.